evm_bridge/src/verifier/batch.ts

```diff
@@ -1,5 +1,5 @@
 import { Scalar } from "o1js";
-import { Verifier, VerifierIndex } from './Verifier.js'
+import { Verifier, VerifierIndex } from './verifier'
```

Comment:
This doesn't fail because it's not called from main, but I think this would fail too

Reply:
solved!